README.md

```diff
@@ -75,34 +76,48 @@ To get redirection to these pages, or redirection to third-party services, it is
 - file: `example/proxy_conf.yml`, example value: `UNKNOW_ERROR_REDIRECT_PAGE: "https://static-contents.example.org/error_page.html"`
 - file: `example/plugins/{backends,frontends}/$filename`, example value: `disco_srv: "https://static-contents.example.org/static/disco.html"`
 
+<hr>
 
-## Docker
+## How to start the environment
 
-<img src="gallery/docker-design.svg" width="512">
+The average time to set up the environment is about 1 hour. This time may vary depending on the machine's resources and the type of network connection.
 
-The official Satosa-Saml2SPID docker image is available at 
-[italia/satosa-saml2spid](https://ghcr.io/italia/satosa-saml2spid).
+> Make sure that in your environment is correcly installed:
+> - a version of Python 3.10 or higher
+> - Git
+> - Docker
 
-Below some quick commands:
+#### STEP 1 - Setup
+please review the following documentation in order to install, configure and run Satosa-Saml2spid
+ [README-SETUP.md](README-Setup.md)
 
-- Install it, execute the following command: `sudo docker pull ghcr.io/italia/satosa-saml2spid:latest`.
-- Build locally the image, execute the following command: `docker build -t satosa-saml2spid .`.
-- Inspect the image content: `docker run -it -v $(pwd)/example:/satosa_proxy --entrypoint sh satosa-saml2spid`.
+#### STEP 2 - Docker Compose
 
+please review the following documentation [Docker-compose](Docker-compose/README.md) in order to create the volumes:
+- satosa-saml2spid_mongodata
+- satosa-saml2spid_nginx_certs
 
-### Docker compose
 
 Satosa-Saml2Spid image is built with production ready logic.
-The docker compose may use the [enviroment variables](#configuration-by-environment-variables) 
+The docker compose may use the [enviroment variables](README-Setup.md#configuration-by-environment-variables) 
 to configure Satosa-Saml2Spid.
 
-See [Docker-compose](Docker-compose) for details.
+<img src="gallery/docker-design.svg" width="512">
+
+The official Satosa-Saml2SPID docker image is available at 
+[italia/satosa-saml2spid](https://ghcr.io/italia/satosa-saml2spid).
+
+Below some quick commands:
 
+- Install it, execute the following command: `sudo docker pull ghcr.io/italia/satosa-saml2spid:latest`.
+- Build locally the image, execute the following command: `docker build -t satosa-saml2spid .`.
+- Inspect the image content: `docker run -it -v $(pwd)/example:/satosa_proxy --entrypoint sh satosa-saml2spid`.
```

Comment:
```suggestion
To install it, you can execute the following command: `sudo docker pull ghcr.io/italia/satosa-saml2spid:latest`.

Otherwise you can build the image executing the following command: `docker build -t satosa-saml2spid .`.

Then you can even inspect the image content, by running the following command: `docker run -it -v $(pwd)/example:/satosa_proxy --entrypoint sh satosa-saml2spid`.